youtube_dl/extractor/vgtv.py

```diff
@@ -11,16 +11,19 @@
 
 
 class VGTVIE(InfoExtractor):
-    IE_DESC = 'VGTV and BTTV'
+    IE_DESC = 'VGTV, BTTV, FTV, Aftenposten and Aftonbladet'
     _VALID_URL = r'''(?x)
-                    (?:
-                        vgtv:|
```

Comment:
Then make it compatible? It's not about using in another extractor but about using as shortcuts by user in the first place. And it obviously should be preserved for backward compatibility